conan/cli/formatters/graph/info_graph_html.py

```diff
@@ -82,11 +82,13 @@
                 // Add error conflict node
                 nodes.push({
                     id: "{{ error["type"] }}",
-                    label: "{{ error["context"].require.ref }}",
+                    label: {{ error["context"].require.ref|string|tojson }},
```

Comment:
Note that the `tojson` function adds proper quoting to the label, so that they are not needed in the js code itself

Reply:
I dont know enough of this lib, but it seems it makes sense 👍